plugins/notification-resources/src/components/NotificationSettings.svelte

```diff
@@ -44,6 +45,19 @@
   })
 
   let group: Ref<NotificationGroup> | undefined = undefined
+  let currentPreferenceGroup: NotificationPreferencesGroup | undefined = undefined
+  
+  client.findAll(notification.class.NotificationPreferencesGroup, {}).then((res) => {
```

Comment:
Please use query instead, since you could have multiple tabs/browsers, so you should receive notifications.

Reply:
I considered this. My intuition was like that: those items are pretty static, they are for preferences items as such:

<img width="1241" alt="Screenshot 2023-09-04 at 09 56 50" src="https://github.com/hcengineering/anticrm/assets/3119549/a606ce38-9b22-4ec9-8a50-8e7ca272bacc">

And they should be pretty static, like we are creating them on workspace setup. Do we really want to use query for such static content too?

Reply:
ok, is for groups.